### PR TITLE
:sparkles: Use Discord avatar

### PR DESCRIPTION
**Pull request by cchampou**
# Show Discord avatar at the top right corner of the dashboard

## Back-end

I'm now storing the Discord ID and the Discord avatar hash.

## Front-end

The page is using the Discord ID and the Discord avatar hash to generate an image URL.

Screenshot 2022-01-10 at 12-25-27 Accountability Dashboard

### CODE REVIEW

**Review by oezguerisbert — APPROVED** [review]
LGTM

File: frontend/src/modules/discord/avatars.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export const generateAvatarUrl = (
+  discordId: User['discordId'],
+  discordAvatar: User['discordAvatar']
+): string => {
+  if (
+    !discordAvatar ||
+    discordAvatar.length === 0 ||
+    !discordId ||
+    discordId.length === 0
+  )
```
Comment (Juliotati) [inline frontend/src/modules/discord/avatars.ts]: Beautiful ✨ 
Just one thing @cchampou , can the logic on the if statement be moved/held in a variable for a better reading experience
`eg.`
```javaScript    
    if (!hasDiscordAvatar)
        return 'https://icon-library.com/images/no-profile-picture-icon/no-profile-picture-icon-15.jpg';
   return `${process.env.REACT_APP_DISCORD_CDN}/avatars/${discordId}/${discordAvatar}.png`;
```

Comment (cchampou) [reply frontend/src/modules/discord/avatars.ts]: Fixed :wink:

**Review by Juliotati — APPROVED** [review]
✨Beautiful!!
Can I cry? @cchampou

**Comment by cchampou** [comment]
> sparklesBeautiful!! Can I cry? @cchampou

Please don't :hugs:

**Pull request merged**